src/test/kotlin/net/atos/zac/mail/MailServiceTest.kt

```diff
@@ -141,9 +135,12 @@ class MailServiceTest : BehaviorSpec({
                         "geen"
                     )
                 }
-                with(mailjetRequestSlot.captured) {
-                    resource shouldBe "send"
-                    JSONObject(body).getJSONArray("Messages").getJSONObject(0).getString("Subject") shouldBe resolvedSubject
+                with(transportSendRequest.captured) {
+                    subject shouldBe resolvedSubject
+                    with((content as MimeMultipart).getBodyPart(0)) {
+                        contentType shouldBe "text/plain"
```

Comment:
This is not "text/html" here? How does this work?

Reply:
This is the original content. There's datahandler that contains what we want this to become.

Changed the test, so hopefully this would be less awkward